.circleci/main.yml

```diff
@@ -1901,6 +1943,17 @@ workflows:
             - sign-off
             - jupyter-extension-docker-build
           filters: *only-release-tags
+      - python-sdk-test-publish:
+          version: ${CIRCLE_TAG:1}
+          requires:
+            - test-python-sdk
+          filters: *only-release-tags
+      - python-sdk-publish:
+          version: ${CIRCLE_TAG:1}
+          requires:
+            - test-python-sdk
```

Comment:
if this is the final publish job lets add `requires sign-off` too. we use that to gate it until we can fully trust this thing.